src/shared/components/TimeSeries.tsx

```diff
@@ -419,7 +419,7 @@ const mstp = (state: AppState, props: OwnProps) => {
   return {
     hasUpdatedTimeRangeInVEO: hasUpdatedTimeRangeInVEO(state),
     isCurrentPageDashboard: isCurrentPageDashboardSelector(state),
-    queryLink: state.links.query.self,
+    queryLink: '/api/v2/query',
```

Comment:
like the PR description says, we're just hardcoding this rather than making a request at the app level for this info